stablehlo/tests/CheckOps.cpp

```diff
@@ -23,6 +23,7 @@ limitations under the License.
 #include "llvm/Support/Path.h"
 #include "mlir/IR/BuiltinTypes.h"
 #include "mlir/Support/DebugStringHelper.h"
+#include "stablehlo/dialect/Base.h"
```

Comment:
Is this needed? Same question for the BUILD file change. Looks like no file change? Also if needed is there a corresponding cmake change?

Reply:
`stablehlo/tests/CheckOps.cpp.inc` needs the declaration for `isValidStablehloQuantizedElementType` which is in Base.h

Will add the cmake change. Earlier it was satisfied indirectly